### PR TITLE
 Person/Location not selected after creation (EXPOSUREAPP-4387)

### PR DESCRIPTION
**Pull request by kolyaopahle**
### Description
This PR allows the contact diary to auto select a Person / Location after it's been added via the bottom sheet

### Steps to reproduce
1. Open a day in the contact diary
2. Create a new Person / Location
3. Check if Person / Location is auto selected

### CODE REVIEW

**Review by ralfgehrer — COMMENTED** [review]
Tested on Pixel (Android 10) and works nicely for locations and persons.

**Review by ralfgehrer — DISMISSED** [review]

File: Corona-Warn-App/src/main/res/navigation/contact_diary_nav_graph.xml [hunk 1]
```diff
@@ -37,23 +37,33 @@
...
+        android:name="de.rki.coronawarnapp.contactdiary.ui.sheets.location.ContactDiaryLocationBottomSheetDialogFragment"
+        android:label="ContactDiaryLocationBottomSheetDialogFragment">
         <argument
             android:name="selectedLocation"
             app:nullable="true"
             android:defaultValue="@null"
             app:argType="de.rki.coronawarnapp.contactdiary.storage.entity.ContactDiaryLocationEntity" />
+        <argument
+            android:name="addedAt"
+            android:defaultValue="null"
```
Comment (chiljamgossow) [inline Corona-Warn-App/src/main/res/navigation/contact_diary_nav_graph.xml:64]: @null

Comment (chiljamgossow) [reply Corona-Warn-App/src/main/res/navigation/contact_diary_nav_graph.xml:64]: I think you are passing the string null here

Comment (kolyaopahle) [reply Corona-Warn-App/src/main/res/navigation/contact_diary_nav_graph.xml:64]: uh yeah, good spot!

**Review by BMItr — APPROVED** [review]
all right

**Review by ralfgehrer — APPROVED** [review]

**Pull request merged**